science-pipelines/science-pipelines-in-notebooks.rst

```diff
@@ -0,0 +1,82 @@
+###########################################################
+Using the LSST Science Pipelines in notebooks (LSST kernel)
+###########################################################
+
+The LSST kernel includes the `LSST Science Pipelines`_, making it a convenient way to work with LSST’s software.
+The version of the LSST Science Pipelines that’s preinstalled corresponds to the :ref:`image you selected at log in <logging-in-image>`.
+
+This page describes how to use and customize the LSST kernel in notebooks.
+
+.. seealso::
+
+   :doc:`science-pipelines-in-terminals`.
+
+.. _lsst-kernel-create:
+
+Create a new notebook with the LSST kernel
+==========================================
+
+1. From the `file browser`_, click the **+** button (or type :kbd:`command`\ -\ :kbd:`shift`\ -\ :kbd:`L`) to open the launcher.
+2. Under the **Notebook** heading, click on the **LSST** icon.
+
+Now the LSST Science Pipelines are available for you to import and use.
+For example, in a notebook cell, import ``lsst.afw`` and check the version:
+
+.. code-block:: python
+
+   from lsst import afw
+   afw.version.__version__
+
+With the LSST kernel you don’t run the ``loadLSST.bash`` and ``setup`` commands that are needed for command-line usage. The LSST kernel does this for you. You can still customize how packages are set up, see :ref:`lsst-kernel-user-setups`.
+
+.. seealso::
+
+   The `JupyterLab Notebooks documentation`_ has more information on creating and using notebooks.
+
+.. _lsst-kernel-switch:
+
+Switch an opened notebook to the LSST kernel
+============================================
+
+You can open existing notebooks by double clicking on their names in the JupyterLab file browser.
+JupyterLab automatically opens notebooks in the same kernel that they were created with.
+If you’re opening an notebooks that wasn’t created in the Notebook Aspect, you may need to switch the kernel to LSST:
+
+1. Look for the kernel name in the notebook’s upper right menu bar. It should read **LSST**. If it is **Python 3** (the default kernel) you’ll need to switch it.
+2. Click on the kernel name and select **LSST** from the menu.
+
+.. _lsst-kernel-user-setups:
+
+Customizing the EUPS package setup (~/notebooks/.user_setups)
+=============================================================
+
+When the LSST kernel starts up, it activates the LSST environment and sets up ``lsst_distrib`` (the top-level EUPS package of the `LSST Science Pipelines`_).
+
+You can customize what packages are set up by creating and editing a :file:`~/notebooks/.user_setups` file.
```

Comment:
I just confirmed that environment variables can be set here too. They are picked up after restarting the kernel. Excellent!

Reply:
Might be good to add that restarting the kernel will pick up the changes.